packages/tools/kda-cli/.eslintrc.cjs

```diff
@@ -4,4 +4,7 @@ require('@rushstack/eslint-config/patch/modern-module-resolution');
 module.exports = {
   extends: ['@kadena-dev/eslint-config/profile/lib'],
   parserOptions: { tsconfigRootDir: __dirname },
+  rules: {
+    'import/no-unresolved': 'off',
+  },
```

Comment:
Why do we have this? If there's a legitimate reason, let's document that in a comment

Reply:
yes, the reason is in kda-cli we are addressing ts files with .js format which I don't know the reason but when I try to remove the format it doesn't build it anymore!
for example this is an import within kda-cli
`import { IAnswers, IQuestion } from '../questions/questions.js';`

even though the extension for questions file is `.ts`

because of that all the imports will have unresolved warning, so I turned it off for this module